### PR TITLE
docs(#88): add troubleshooting section

### LINKED ISSUE #88: Unable to build with EAS on Expo SDK 46
### Environment

Library version: 2.4.6
eas-cli: 2.1.0
Expo SDK: 46.0.10
Workflow: Managed

### Affected platforms

- [x] Android
- [ ] iOS

### Current behavior

When I decide to build my app with EAS Build for Android I get this. I think this error comes from the Kotlin version and all the errors I get point to this package. I really hope I'm not wrong.

Capture d’écran (204)

### Expected behavior

I expect that the build will success like on IOS 

### Reproduction

I think install this package on an Expo managed App wth SDK 46 and build it will create the same result on Android

### PR DESCRIPTION
**Pull request by Randall71**
This pull request resolves #88 

**Description**

Add a troubleshooting section on the documentation to tackle building Android with this library

**Affected platforms**

- [x] Android
- [ ] iOS

### CODE REVIEW

File: docs/docs/guides/INSTALLATION.mdx [hunk 1]
```diff
@@ -26,3 +26,46 @@ npx pod-install
...
+
+```json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-build-properties",
+        {
+          "android": {
+            "kotlinVersion": "1.16.0"
```
Comment (mateusz1913) [inline docs/docs/guides/INSTALLATION.mdx:64]: Probably you wanted 1.6.10 here

Comment (Randall71) [reply docs/docs/guides/INSTALLATION.mdx]: Yeah sorry, I'll fix that soon

Comment (Randall71) [reply docs/docs/guides/INSTALLATION.mdx]: @mateusz1913 i think all is good now.

**Review by mateusz1913 — APPROVED** [review]
Awesome, thank you for the contribution 🚀

**Pull request merged**